CDCgov/prime-simplereport
# Fix deploys/slot swap

# DEVOPS PULL REQUEST

## Related Issue

- resolves #5934 

## Changes Proposed

- By not setting the application_stack on the web app, we can complete the deployment by swapping from the staging slot.

## Additional Information

- I'm not feeling so good that this was broken in the way it was, but I'm proud that our team norms and pipelines prevented us from deploying a broken build.

## Testing

- How should reviewers verify this PR?

- Deploy to your favorite environment and test slot swaps using the buttons in the `Deployment slots` blade in the API app service you're testing. ( You can use the `alis/5934` branch and click into the `Patients` list in the main header, you'll see that it's broken )
- In the Azure portal, to validate what tag is running in which slot, you can select the slot you want to inspect and visit the `Deployment Center` blade and check the `Tag` attribute on that page.
- You can also check the actuator endpoints ( staging slot: `/staging/api/actuator/info`) and ( prod slot: `/api/actuator/info` )

## Checklist for Primary Reviewer
### Infrastructure
- [ ] Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!

### Security

### Cloud
- [ ] Oncall has been notified if this change is going in after-hours
- [ ] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### Documentation
